src/test/java/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,152 @@
+package subway.line;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.TestConstructor;
+import org.springframework.test.context.TestConstructor.AutowireMode;
+import subway.domain.LineRepository;
+import subway.domain.StationRepository;
+
+@DisplayName("지하철 노선 관련 기능")
+@TestConstructor(autowireMode = AutowireMode.ALL)
+@SpringBootTest(webEnvironment = WebEnvironment.DEFINED_PORT)
+public class LineAcceptanceTest {
+
+    @LocalServerPort
+    private int port;
+    private Long upStationId;
+    private Long downStationId;
+
+    private final LineRestAssured lineRestAssured;
+    private final LineAssert lineAssert;
+
+    private final StationRepository stationRepository;
+    private final LineRepository lineRepository;
+
+    public LineAcceptanceTest(
+            final StationRepository stationRepository,
+            final LineRepository lineRepository
+    ) {
+        this.lineRestAssured = new LineRestAssured();
+        this.lineAssert = new LineAssert();
+        this.stationRepository = stationRepository;
+        this.lineRepository = lineRepository;
+    }
+
+    @BeforeEach
+    void setUp() {
+        RestAssured.port = port;
+        stationRepository.truncateTableStation();
+        lineRepository.deleteAllAndRestartId();
+        this.upStationId = createStationResponseBy("강남역").jsonPath().getLong("id");
+        this.downStationId = createStationResponseBy("양재역").jsonPath().getLong("id");
+    }
+
+    private ExtractableResponse<Response> createStationResponseBy(final String stationName) {
+        Map<String, String> params = new HashMap<>();
+        params.put("name", stationName);
+
+        return RestAssured
+                .given().log().all()
+                .body(params)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .post("/stations")
+                .then().log().all()
+                .statusCode(HttpStatus.CREATED.value())
+                .extract();
+    }
+
+    @DisplayName("지하철 노선 생성한다.")
+    @Test
+    void createLine() {
+        // when
+        String name = "신분당선";
+        String color = "bg-red-600";
+        int distance = 10;
+
+        lineRestAssured.createLine(name, color, upStationId, downStationId, distance);
```

Comment:
static method 라서 객체 생성없이 다음과 같이도 가능합니다 😄 
```suggestion
        LineRestAssured.createLine(name, color, upStationId, downStationId, distance);
```

Reply:
static을 다 제거하려고 했는데 놓쳤네요 😂
static 메소드는 객체 생성 없이 다른 곳에서도 쉽게 사용할 수 있어 캡슐화를 저하시킨다고 생각합니다.
그래서 객체 생성 후 사용할 수 있도록 만들려고 하는데 어떻게 생각하시나요?

Reply:
static 메소드 말씀하신 것 처럼 객체지향적이지 않은 구조이지만 그렇다고 해서 무조건 지양해야 하는 부분은 아닐 것 같아요.
 staitc 메소드는 객체화 하여 사용하는 것 보다 접근이 훨씬 용이하다는 장점이 있을 것 같은데요,
상태값을 가질 필요가 없거나, 공통으로 사용할만한 유틸성 클래스에서는 static 메서드를 사용해도 무방하다고 생각합니다.

다만 static 메소드를 만드실 때 객체지향적인 원칙들에 대해 한번 고민해 보시면 좋을 것 같아요. 😄